app/models/location.rb

```diff
@@ -40,7 +40,7 @@ def num_checkins
 
   def completed?
     num_teams = Team.playing.count
-    num_checkins == num_teams
+    num_checkins >= num_teams
```

Comment:
Ahhh, that totally makes sense.

What I'd probably do is make `num_checkins` exclude checkins from teams that are not currently active. Then the counts should line up.